dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleMetadataConnector.java

```diff
@@ -127,8 +127,10 @@ public void addTasksTo(@Nonnull List<? super Task<?>> out, @Nonnull ConnectorArg
     String whereCondOwner = ownerInList == null ? "" : " WHERE OWNER IN " + ownerInList;
     String whereCondTableOwner = ownerInList == null ? "" : " WHERE TABLE_OWNER IN " + ownerInList;
     String whereCondSequenceNotSystemGeneratedAndOwner =
-        ownerInList == null ? " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$\\_%' ESCAPE '\\'"
-            : " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$\\_%' ESCAPE '\\' AND SEQUENCE_OWNER IN " + ownerInList;
+        ownerInList == null
+            ? " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$\\_%' ESCAPE '\\'"
```

Comment:
nit: since both branches have the same prefix, wouldn't it make sense to move the ternary operation later to underline what is common and what is different?
" WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$\\_%' ESCAPE '\\'" + (ownerInList == null ? "" : " AND SEQUENCE_OWNER IN " + ownerInList)

Reply:
I see your point, but ... this was a line that was exceeding maximum line length,
I don't even know how it got in the repo (I did not produce it - I got it after a `git pull`).
The line was causing my gradle build to fail issuing the usual suggestion to run `./gradlew :dumper:app:spotlessApply` which I did.